metadata.json

```diff
@@ -55,10 +90,17 @@
     },
     {
       "operatingsystem": "AIX",
-      "operatingsystemrelease": ["5.3", "6.1", "7.1"]
+      "operatingsystemrelease": [
+        "5.3",
+        "6.1",
+        "7.1"
+      ]
     }
   ],
-  "pdk-version": "1.16.0",
-  "template-url": "pdk-default#1.16.0",
-  "template-ref": "tags/1.16.0-0-gaf44904"
+  "requirements": [
+    {
+      "name": "puppet",
+      "version_requirement": ">= 7.24 < 9.0.0"
+    }
+  ]
```

Comment:
These are new compared to the the 1.6.1 release of this module. 1.6.2 pulled because PDK added this during the build, except with `">= 6.21.0 < 8.0.0"` as the version constraints, which were clearly wrong.

Reply:
Do we have a way of confirming the module is compatible with Puppet 8?  I'm not aware of all the differences.

Reply:
It is currently being shipped in versions of PE that include Puppet 8. And I think the previous change to fix the `File.exist?` issue was a direct result of people using it with Puppet 8/Ruby 3 and hitting errors. Those were resolved once we released 1.6.1. See https://github.com/puppetlabs/puppetlabs-cd4pe_jobs/blob/master/CHANGELOG.md.

The previous version constraints generated by PDK were due to me using an old version of PDK, nothing to do with the contents of this module.

Reply:
1.6.1 and previous did not display any constraints on the Forge. However, it is recommended to display them by the module community. They are non-binding.

Reply:
Ok, seems all good to me then!